locales/de/subtitles/lidia-valeriani/lidia-valeriani-01.yml

```diff
@@ -128,8 +128,8 @@ de:
           s: 24
           from: 00:01:53,11
           to: 00:02:02,28
-          text: He would have never enlisted us in the “Piccole Italiane”, or the
-            boys in the “Balilla”.
+          text: Niemals hätte er uns bei den „Piccole Italiane“ oder die Jungen
+            bei den „Balilla“ angemeldet.
```

Comment:
I suppose `Jungen` is the word Lidia would have used. IMHO `Jungs` sounds too modern.
@RoteReiter What is your opinion?